ig/input/fsh/VaccinationScheme.fsh

```diff
@@ -6,9 +6,56 @@ Description: """
   VaccinationScheme models a process consisting of immunizations through wich a patient can reach immunity against a target
   disease.
 """
```

Comment:
Can we add the MS flag to `code` so it shows up in the must-support snapshot table?

```suggestion
"""
* code 1..1 MS
```